### PR TITLE
fix(learn): avoid conflict with spring forward EST to EDT

### PR DESCRIPTION
**Pull request by jeremyagray**
March 8, 2009 is the weekend of spring forward to EDT in the US.  This change moves the example date and test back one day to add twelve hours to a time on March 6, 2009, resulting in a time on March 7, 2009.

Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] All the files I changed are in the same world language, for example: only English changes, or only Chinese changes, etc.

Closes #XXXXX

The example and test four in Rosetta Code:  Date Manipulation ignore the fact that the time changes from EST to EDT on March 8, 2009, which is the target date of the example and test.  This moves the date back one day to avoid the problem.

### CODE REVIEW

**Review by naomi-lgbt — APPROVED** [review]
LGTM~! 🥳 Nice catch!

Congratulations on submitting your first pull request to freeCodeCamp! As a heads up, our workflow requires at least one additional approval before this can be merged. 🙂

**Review by raisedadead — APPROVED** [review]

**Pull request merged**
